js/main.js

```diff
@@ -50,3 +57,199 @@ function submitForm(event) {
 }
 
 $triviaForm.addEventListener('submit', submitForm);
+
+function seeQuestions(triviaObj) {
+  $setUpContainer.setAttribute('class', 'hidden');
+  $questionContainer.setAttribute('class', 'question-container');
+
+  dataModel.results = triviaObj.results;
+
+  var results = dataModel.results;
+
+  var correctAnswer = results[0].correct_answer;
+
+  dataModel.correctAnswer = correctAnswer;
+
+  var choicesArr = [correctAnswer, results[0].incorrect_answers[0], results[0].incorrect_answers[1], results[0].incorrect_answers[2]];
+  var num = 3;
+
+  var $questionDiv = document.createElement('div');
+  $questionDiv.setAttribute('class', 'row');
+  $questionContainer.appendChild($questionDiv);
+
+  var $h1 = document.createElement('h1');
+  $h1.setAttribute('class', 'question');
+  $h1.textContent = 'Question: ' + results[0].question;
+  $questionDiv.appendChild($h1);
+
+  var $choiceDiv = document.createElement('div');
+  $choiceDiv.setAttribute('class', 'row flex-wrap margin-top-head');
+  $questionContainer.appendChild($choiceDiv);
+
+  var $questionDiv1 = document.createElement('div');
+  var $questionDiv2 = document.createElement('div');
+  var $questionDiv3 = document.createElement('div');
+  var $questionDiv4 = document.createElement('div');
+
+  $questionDiv1.setAttribute('class', 'column-half');
+  $questionDiv2.setAttribute('class', 'column-half');
+  $questionDiv3.setAttribute('class', 'column-half');
+  $questionDiv4.setAttribute('class', 'column-half');
+
+  $choiceDiv.appendChild($questionDiv1);
+  $choiceDiv.appendChild($questionDiv2);
+  $choiceDiv.appendChild($questionDiv3);
+  $choiceDiv.appendChild($questionDiv4);
+
+  var $p1 = document.createElement('p');
+  var $p2 = document.createElement('p');
+  var $p3 = document.createElement('p');
+  var $p4 = document.createElement('p');
+
+  $p1.setAttribute('class', 'choice');
+  $p2.setAttribute('class', 'choice');
+  $p3.setAttribute('class', 'choice');
+  $p4.setAttribute('class', 'choice');
+
+  $p1.setAttribute('id', 'one');
+  $p2.setAttribute('id', 'two');
+  $p3.setAttribute('id', 'three');
+  $p4.setAttribute('id', 'four');
+
+  $questionDiv1.appendChild($p1);
+  $questionDiv2.appendChild($p2);
+  $questionDiv3.appendChild($p3);
+  $questionDiv4.appendChild($p4);
+
+  var $choice1 = document.createElement('span');
+  var $choice2 = document.createElement('span');
+  var $choice3 = document.createElement('span');
+  var $choice4 = document.createElement('span');
+
+  var randomInt = Math.floor(Math.random() * num);
+  if (choicesArr[randomInt] === correctAnswer) {
+    $choice1.textContent = choicesArr[randomInt];
+    choicesArr.splice(randomInt, 1);
+    num--;
+
+    dataModel.correctChoiceDom = $p1;
+  } else {
+    $choice1.textContent = choicesArr[randomInt];
+    choicesArr.splice(randomInt, 1);
+    num--;
+
+  }
```

Comment:
You've repeated this same behavior four times in this function, you should make this into a separate function definition you can just call 4 times in your code.